guardian/testapp/testsettings.py

```diff
@@ -5,7 +5,7 @@
 
 env = environ.Env()
 
-DEBUG = False
+DEBUG = True
```

Comment:
```suggestion
DEBUG = False
```
This is not intended and has to be reverted